monkey/monkey_island/cc/services/reporting/issue_processing/exploit_processing/exploiter_descriptor_enum.py

```diff
@@ -0,0 +1,32 @@
+from dataclasses import dataclass
+from enum import Enum
+from typing import Type
+
+from monkey_island.cc.services.reporting.issue_processing.exploit_processing import CredExploitProcessor
+from monkey_island.cc.services.reporting.issue_processing.exploit_processing import ExploitProcessor
+from monkey_island.cc.services.reporting.issue_processing.exploit_processing import \
+    ShellShockExploitProcessor
+
+
+@dataclass
+class ExploiterDescriptor:
+    # Must match with class names of exploiters in Infection Monkey code
+    class_name: str
+    display_name: str
+    processor: Type[ExploitProcessor]
+
+
+class ExploiterDescriptorEnum(Enum):
```

Comment:
Why is this an enum? What functionality of enum are you using?

Reply:
Why not? We're using iteration in `report.py`. Also, importing enum is only one import statement vs importing consts. Also, class name gives additional context, which improves readability. What do you think would be a better data structure?